### ISSUE TITLE
Lightning integration

### ISSUE THREAD

**Issue by nikk-nikaznan** [SainsburyWellcomeCentre/crabs-exploration#152]
Some issues for the next TI.

- The output of model.eval() not returning loss, need to write some loss function for validation. For now using precision and recall values.
- Find the way to return the output from the model so we can use to save the images with the bounding boxes

**Comment by nikk-nikaznan** [comment]
Duplicate #159
